vaadin-upload-flow-parent/vaadin-upload-flow-integration-tests/src/test/java/com/vaadin/flow/component/upload/tests/UploadI18nIT.java

```diff
@@ -52,31 +42,8 @@ public void testFullI18nShouldAffectLabels() {
                 dropLabel.getText());
     }
 
-    /**
-     * Verifies that every single translation provided by the UploadI18N
-     * instance is set in the web component.
-     *
-     * Testing internals here, in favour of setting up the web component with
-     * files/event handlers for every possible state.
-     */
     @Test
-    public void testFullI18nShouldOverrideCompleteConfigurationInWebComponentProperty() {
```

Comment:
These tests were checking that the nested I18N structure was properly merged by the `executeJs` call that has now been removed. The equivalent would now be to check the internal `__effectiveI18n`.

However, since the merging logic is part of the web component, I think it's better to remove this. The web components could use some better test coverage for partial I18n, but that's a separate issue.